src/main/java/mate/academy/spring/controller/MovieController.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.MovieRequestDto;
+import mate.academy.spring.dto.MovieResponseDto;
+import mate.academy.spring.mapper.MovieMapper;
+import mate.academy.spring.service.MovieService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movies")
+public class MovieController {
+    private MovieService movieService;
+    private MovieMapper movieMapper;
+
+    public MovieController(MovieService movieService,
+                           MovieMapper movieMapper) {
+        this.movieService = movieService;
+        this.movieMapper = movieMapper;
+    }
```

Comment:
You should add @Autowired here or remove it from each controller in order your code to be consistent
```suggestion
    @Autowired
    public MovieController(MovieService movieService,
                           MovieMapper movieMapper) {
        this.movieService = movieService;
        this.movieMapper = movieMapper;
    }
```